Notifier-service/src/services/conference.ts

```diff
@@ -31,14 +31,59 @@ export class ConferenceServiceI extends ConferenceService{
     }
 
 
-    async getConferences(offset:Number , count:Number):Promise<Conference[]>{
-        return Promise.resolve([])
+    async getConferences(offset:number , count:number):Promise<Conference[]>{
+        this.logger.debug("getConferences invoked")
+        return new Promise<Conference[] | []>((resolve , reject) => {
+            this.conferenceModel.getConferences(offset, count).then( value => {
+                if(value == []){
+                    resolve([])
+                }
+                else{
+                    let result = Object.assign({}, value);
```

Comment:
* You can resolve to ConferenceDocument.getObject() method also, Its provided by the document object returned by the model api.

Reply:
getObject() method only works for ConferenceDocument but here we are getting array of conferenceDocument in response that's why i have resolved it this way.

Reply:
You can always map them. Problem is. The document contains extra stuff, which does not come up when you  `console.log` because the string method is overriden to show the actual data from db. So better to map over and extract.